wordcloud/wordcloud_cli.py

```diff
@@ -12,12 +12,15 @@
 import io
 import re
 import argparse
-import wordcloud as wc
-import numpy as np
+
 from PIL import Image
 
+import numpy as np
+
```

Comment:
I feel like adding so many newlines decreases readability. What linter asks that?

Reply:
pep8 says:
```
You should put a blank line between each group of imports.
```

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
pe8 mentions three groups, though. and there are 9 here now, right?

Reply:
yes,
i seperate "import ..." statement and "from ... import ..." statement for show the difference between modules and functions or variables clearly
in my opinion, it's better to use only "from ... import ..." statement to know what functions or variables used and when an api changed, you can just use easily "from ... import ... as ..." statement